pkg/client/clientcmd/client_config.go

```diff
@@ -250,6 +250,9 @@ func (config DirectClientConfig) Namespace() (string, error) {
 	if len(configContext.Namespace) == 0 {
 		return api.NamespaceDefault, nil
 	}
+	if configContext.Namespace == "*" {
```

Comment:
Thank you for kindly clarification. Now --all-namespaces flag is added, it is handled in resource builder.